astroid/util.py

```diff
@@ -22,16 +32,18 @@ def lazy_import(module_name):
     )
 
 
-@object.__new__
-class Uninferable:
-    """Special inference object, which is returned when inference fails."""
+class UninferableBase:
+    """Special inference object, which is returned when inference fails.
 
-    def __repr__(self):
+    This is meant to be used as a singleton. Use util.Uninferable to access it.
```

Comment:
Would we rather point to `astroid.Uninferable`, which is where `pylint` usually accesses it?